tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -985,15 +985,17 @@ public function test_next_tag_and_set_attribute_in_a_loop_update_all_tags_in_the
 	 * Removing an attribute that's listed many times, e.g. `<div id="a" id="b" />` should remove
 	 * all its instances and output just `<div />`.
 	 *
-	 * Today, however, WP_HTML_Tag_Processor only removes the first such attribute. It seems like a corner case
+	 * <del>Today, however, WP_HTML_Tag_Processor only removes the first such attribute. It seems like a corner case
```

Comment:
Why not remove this text entirely?

Reply:
Yeah probably it's not something people will want to keep around, but I found that the idea was useful to demonstrate in the code how this has changed, or why it's different now.

Maybe pre-merge I'll remove it if someone also feels that way.